src/main/java/com/scireum/App.java

```diff
@@ -67,20 +68,22 @@ public static void main(String[] args) {
             verifyCommandLine(args);
             createScriptingEngine();
             evalDonkeyLibrary();
-            loadAndExecuteScript(args[0]);
+            loadAndExecuteScript(args[0], Arrays.copyOfRange(args, 1, args.length));
         } catch (Exception t) {
             Exceptions.handle(t);
         } finally {
             Sirius.stop();
         }
     }
 
-    private static void loadAndExecuteScript(String arg) throws ScriptException, FileNotFoundException {
-        engine.put(ScriptEngine.FILENAME, arg);
-        LOG.INFO("Executing '%s'...", arg);
+    private static void loadAndExecuteScript(String filename, String[] args)
+            throws ScriptException, FileNotFoundException {
+        engine.put(ScriptEngine.FILENAME, filename);
+        engine.getContext().setAttribute("args", args, ScriptContext.ENGINE_SCOPE);
```

Comment:
Shouldn't we wrap this in "Values"
```suggestion
        engine.getContext().setAttribute("args", Values.of(args), ScriptContext.ENGINE_SCOPE);
```